policyengine_canada/variables/gov/provinces/qc/tax/income/credits/work_premium/qc_work_premium_credit.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_canada.model_api import *
+
+
+class qc_work_premium_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec work premium tax credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        qc_general_work_premium_amount = household(
+            "qc_general_work_premium_amount", period
+        )
+        qc_adapted_work_premium_amount = household(
+            "qc_adapted_work_premium_amount", period
+        )
+
+        # If you are entitled to both the work premium and the adapted work premium, you will receive the greater of the two
+        credit_amount = max_(
```

Comment:
Then rename this `qc_work_premium_credit_base` and make `qc_work_premium_credit` an `adds = ["qc_work_premium_credit_base", "qc_work_premium_credit_supplement"]` (in general simple sums are helpful to separate)
```suggestion
        return max_(
```